samples/bluetooth/ipsp/README.rst

```diff
@@ -27,14 +27,132 @@ following commands:
    $ modprobe bluetooth_6lowpan
    $ echo 1 > /sys/kernel/debug/bluetooth/6lowpan_enable
 
-Then to connect:
+If you connected your board to a UART console, you will see an output similar to
+(may vary slightly by application and Zephyr versions):
+
+.. code-block:: console
+
+   [bt] [WRN] set_static_addr: Using temporary static random address
+   [bt] [INF] show_dev_info: Identity: cb:af:14:57:d8:6e (random)
+   [bt] [INF] show_dev_info: HCI: version 5.0 (0x09) revision 0x0000, manufacturer 0xffff
+   [bt] [INF] show_dev_info: LMP: version 5.0 (0x09) subver 0xffff
+   [bt] [WRN] bt_pub_key_gen: ECC HCI commands not available
+   [ipsp] [INF] init_app: Run IPSP sample
+   [ipsp] [INF] listen: Starting to wait
+
+The output above provides a BLE addressed assigned to your board (for the
+current session, will change on the next run).
+
+Alternatively, you may scan for your board on the host (this usually should
+be run as root, and you will need to break it manually):
+
+.. code-block:: console
+
+   # hcitool lescan
+   LE Scan ...
+   CB:AF:14:57:D8:6E (unknown)
+   CB:AF:14:57:D8:6E Test IPSP node
+
+After you found out BLE address, connect to the board:
 
 .. code-block:: console
 
    $ echo "connect <bdaddr> <type>" > /sys/kernel/debug/bluetooth/6lowpan_control
 
-Once connected a dedicated interface will be created, usually bt0, which can
-then be used as following:
+Where ``<bdaddr>`` is BLE address and ``<type>`` is BLE address type, 1 for public
+address and 2 for static random address. As you can see from the IPSP sample
+output above, it uses the latter. So, with the sample output above, the command
+will be:
+
+.. code-block:: console
+
+   $ echo "connect CB:AF:14:57:D8:6E 2" > /sys/kernel/debug/bluetooth/6lowpan_control
+
+Once connected a dedicated interface will be created, usually bt0. You can verify this
+with the following command:
+
+.. code-block:: console
+
+   # ifconfig
+   bt0       Link encap:UNSPEC  HWaddr F8-2F-A8-FF-FE-EB-6D-8C-00-00-00-00-00-00-00-00
+             inet6 addr: fe80::fa2f:a8ff:feeb:6d8c/64 Scope:Link
+             UP POINTOPOINT RUNNING MULTICAST  MTU:1280  Metric:1
+             RX packets:2 errors:0 dropped:3 overruns:0 frame:0
+             TX packets:6 errors:0 dropped:0 overruns:0 carrier:0
+             collisions:0 txqueuelen:1000
+             RX bytes:92 (92.0 B)  TX bytes:233 (233.0 B)
+
+As can be seen from the output, only a link-local IPv6 address was assigned
+to the interface.
+
+At this point, you can already test IPv6 connectivity (and discover your board's IPv6
+address) by pinging "All local-link nodes" IPv6 address:
+
+.. code-block:: console
+
+   # ping6 -I bt0 ff02::1
+   PING ff02::1(ff02::1) from fe80::fa54:a8ff:feeb:218f bt0: 56 data bytes
+   64 bytes from fe80::fa54:a8ff:feeb:218f: icmp_seq=1 ttl=64 time=0.088 ms
+   64 bytes from fe80::c9af:14ff:fe57:d86e: icmp_seq=1 ttl=64 time=285 ms (DUP!)
+
+For each ping packet, both your host and BLE board send reply, so you can see
```

Comment:
change to:

     both your host and the BLE board send a reply; you can see the
     board's reply market as `(DUP!)`